.github/workflows/deploy_agent.yml

```diff
@@ -10,33 +10,40 @@ on:
         description: 'Run ID of the Release Workflow (all_solutions.yml) that was triggered by creating a Release in GitHub.  ID can be found in URL for run.'
         required: true
       deploy:
-        description: 'If "true", deploy the artifacts. If "false", do everything except deploy.'
+        type: boolean
+        description: Deploy Build Artifacts. (If not selected, do everything except deploy.)
         required: true
-        default: 'false'
+        default: true
       downloadsite:
-        description: 'If "true", will run the deploy-downloadsite job. If "false", will not.'
+        type: boolean
+        description: Deploy MSI / ZIP / Tarball to the Download Site
         required: true
-        default: 'true'
+        default: true
       nuget:
-        description: 'If "true", will run the deploy-nuget job. If "false", will not.'
+        type: boolean
+        description: Deploy Nuget Packages
         required: true
-        default: 'true'
+        default: true
       linux:
-        description: 'If "true", will run the deploy-linux job. If "false", will not.'
+        type: boolean
+        description: Deploy APT / YUM Packages
         required: true
-        default: 'true'
+        default: true
       linux-deploy-to-production:
-        description: 'If "true", will deploy Linux packages to the real apt/yum.newrelic.com. If "false", will deploy to a test repository.'
+        type: boolean
+        description: Deploy APT / YUM Packages to Production Repository. (If not selected, will deploy to a test respository.)
         required: true
-        default: 'false'
+        default: true
```

Comment:
This default used to be `false`, is there a reason we are changing it to `true`?

Reply:
Same reasoning as for the other setting that was twiddled.